src/views/RealTimeEvents/blocks/content-filter-block.vue

```diff
@@ -71,6 +75,7 @@
           icon-pos="right"
           icon="pi pi-external-link"
           label="Open in GraphQL Playground"
+          @click="props.playgroundOpener()"
```

Comment:
Sugiro importar o método `playgroundOpener` diretamente dentro do component `content-filter-block`, eliminando a necessidade de repassar essa prop por múltiplos níveis.
```suggestion
          @click="props.playgroundOpener"
```